octopusdeploy/templates/pvc.yaml

```diff
@@ -0,0 +1,41 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: repository-claim
+spec:
+  # If you are using one replica, ensure the accessMode value is -ReadWriteOnce
+  # If you have more than one replica, ensure the accessMode value is -ReadWriteMany
+  accessModes:
+    - ReadWriteMany
```

Comment:
`ReadWriteMany` access mode is quite difficult to set up in most environments. When I last looked GKE and EKS both require NFS shares to be built first, and some additional configuration to enable this access mode in their clusters. 

The previous logic was to use `ReadWriteOnce` when the replica count was set to 1, which allowed a single node cluster to be spun up quickly with no additional configuration. I think forcing the use of `ReadWriteMany` will make this chart hard to use for quick testing.

Reply:
Yeah, that's a really good point. We can default it to `ReadWriteOne` and then users can change it based on their environment if they, for example, decide to have NFS shares for EKS.

Done